pkgs/tools/misc/didyoumean/default.nix

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Please run `nixpkgs-fmt`

Reply:
Heh. I did... but then I ran Alejandra as I was curious what it would do, and applied one of the changes it suggested.

I'll stick with `nixpkgs-fmt` until the discussion over formatters dies down a bit.

Reply:
I actually liked how it was before, but ya know, tradition consistency blah blah..

Reply:
Yes, I totally agree, I had the same thought. 
"This looks nice but I feel like it doesn't fit for some reason..."